interfaces/Config/templates/config_special.tmpl

```diff
@@ -51,7 +52,25 @@
                 </fieldset>
             </div><!-- /col1 -->
         </div><!-- /section -->
+
+        <div class="section">
+            <div class="col2">
+                <h3>$T('title-config_lock') <a href="$helpuri$help_uri" target="_blank"><span class="glyphicon glyphicon-question-sign"></span></a></h3>
+                <p>$T('lead-config_lock')</p>
+            </div><!-- /col2 -->
+            <div class="col1">
+                <fieldset>
+                    <div class="field-pair">
+                        <label class="config wide" for="config_lock">
+                            config_lock ( <span class="path">off</span> )
+                        </label>
+                        <span class="desc">$T('explain-config_lock')</span>
```

Comment:
Why does `config_lock` need to be separate?

Reply:
Because unlike all the other settings it can can't be set here at all. So this is just for documentary purposes.

Reply:
Let's leave it out then :)